2024-02-19 00:06:07,935 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 00:06:07,935 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Configure stats pid to 25856
2024-02-19 00:06:07,935 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-19 00:06:07,936 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-19 00:06:07,936 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 00:06:07,936 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-19 00:06:07,937 INFO    MainThread:25856 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_000607-wzmf1wvx\logs\debug.log
2024-02-19 00:06:07,937 INFO    MainThread:25856 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_000607-wzmf1wvx\logs\debug-internal.log
2024-02-19 00:06:07,937 INFO    MainThread:25856 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000246F7F64CD0>
2024-02-19 00:06:07,937 INFO    MainThread:25856 [wandb_init.py:init():566] calling init triggers
2024-02-19 00:06:07,937 INFO    MainThread:25856 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'min_samples': 5, 'max_depth': 41}
2024-02-19 00:06:07,938 INFO    MainThread:25856 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 8wstcap7
2024-02-19 00:06:07,939 INFO    MainThread:25856 [wandb_run.py:_finish():1970] finishing run mc0c0z/DecisionTree/8wstcap7
2024-02-19 00:06:07,939 INFO    MainThread:25856 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-19 00:06:07,939 INFO    MainThread:25856 [wandb_run.py:_restore():2193] restore
2024-02-19 00:06:07,939 INFO    MainThread:25856 [wandb_run.py:_restore():2199] restore done
2024-02-19 00:06:11,176 INFO    MainThread:25856 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-19 00:06:11,176 INFO    MainThread:25856 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-19 00:06:11,183 INFO    MainThread:25856 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-02-19 00:06:11,287 INFO    MainThread:25856 [wandb_init.py:init():616] starting backend
2024-02-19 00:06:11,288 INFO    MainThread:25856 [wandb_init.py:init():620] setting up manager
2024-02-19 00:06:11,289 INFO    MainThread:25856 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-19 00:06:11,289 INFO    MainThread:25856 [wandb_init.py:init():628] backend started and connected
2024-02-19 00:06:11,295 INFO    MainThread:25856 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-19 00:06:11,296 INFO    MainThread:25856 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-19 00:06:11,296 INFO    MainThread:25856 [wandb_init.py:init():720] updated telemetry
2024-02-19 00:06:11,297 INFO    MainThread:25856 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-19 00:06:12,784 INFO    MainThread:25856 [wandb_run.py:_on_init():2262] communicating current version
2024-02-19 00:06:12,969 INFO    MainThread:25856 [wandb_run.py:_on_init():2271] got version response 
2024-02-19 00:06:12,969 INFO    MainThread:25856 [wandb_init.py:init():804] starting run threads in backend
2024-02-19 00:06:13,050 INFO    MainThread:25856 [wandb_run.py:_console_start():2241] atexit reg
2024-02-19 00:06:13,051 INFO    MainThread:25856 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-19 00:06:13,051 INFO    MainThread:25856 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-19 00:06:13,051 INFO    MainThread:25856 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-19 00:06:13,051 INFO    MainThread:25856 [wandb_init.py:init():847] run started, returning control to user process
2024-02-19 00:06:38,478 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 00:06:38,478 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Configure stats pid to 25856
2024-02-19 00:06:38,478 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-19 00:06:38,479 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-19 00:06:38,479 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 00:06:38,479 INFO    MainThread:25856 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-19 00:06:38,480 INFO    MainThread:25856 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_000638-m5qgqcib\logs\debug.log
2024-02-19 00:06:38,480 INFO    MainThread:25856 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_000638-m5qgqcib\logs\debug-internal.log
2024-02-19 00:06:38,480 INFO    MainThread:25856 [wandb_init.py:init():566] calling init triggers
2024-02-19 00:06:38,480 INFO    MainThread:25856 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'min_samples': 5, 'max_depth': 46}
2024-02-19 00:06:38,480 INFO    MainThread:25856 [wandb_init.py:init():591] re-initializing run, found existing run on stack: wzmf1wvx
2024-02-19 00:06:38,482 INFO    MainThread:25856 [wandb_run.py:_finish():1970] finishing run mc0c0z/DecisionTree/wzmf1wvx
2024-02-19 00:06:38,482 INFO    MainThread:25856 [jupyter.py:save_history():445] not saving jupyter history
2024-02-19 00:06:38,482 INFO    MainThread:25856 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-19 00:06:38,482 INFO    MainThread:25856 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-19 00:06:38,483 INFO    MainThread:25856 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-19 00:06:38,483 INFO    MainThread:25856 [wandb_run.py:_restore():2193] restore
2024-02-19 00:06:38,483 INFO    MainThread:25856 [wandb_run.py:_restore():2199] restore done
2024-02-19 00:06:41,637 INFO    MainThread:25856 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-19 00:06:41,637 INFO    MainThread:25856 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-19 00:06:41,643 INFO    MainThread:25856 [wandb_run.py:_footer_sync_info():3825] logging synced files
